apps/settings/test/unit/panels/improve_browser_os/panel_test.js

```diff
@@ -63,7 +63,8 @@ suite('Improve browser os panel > ', function() {
   suite('share performance data toggle', function() {
     test('the toggle should be enabled when not dog fooding', function(done) {
       MockSettingsCache._settings = {
-        'debug.performance_data.dogfooding': null
+        'debug.performance_data.dogfooding': null,
+        'metrics.selectedMetrics.level': 'Basic'
       };
       panel.init(document.body).then(() => {
         var toggle = document.querySelector(
```

Comment:
You need add mozSettings mock in suiteSetup/suiteTeardown since you call the mozSettings api directly. It solves `mozSettings not found` issue.

```
realSettings = window.navigator.mozSettings;
window.navigator.mozSettings = MockNavigatorSettings;
```